contrib/checkbox-provider-ce-oem/units/strict-confinement/thermal-strict-confinement.pxu

```diff
@@ -0,0 +1,22 @@
+unit: template
+template-engine: jinja2
+template-resource: thermal_zones
+template-id: strict-confine/temperature-test
+id: strict-confine/temperature_{{ name }}_{{ type }}
+_summary: Check Thermal temperature of {{ name }} - {{ type }}
+_description:
+    Test a thermal temperature for {{ name }} - {{ type }}.
+category_id: thermal
+plugin: shell
+estimated_duration: 5m
+flags: also-after-suspend
+command:
+    node="/sys/class/thermal/{{ name }}/temp"
+    echo "Starting stress-ng in 10s"
+    sleep 10
+    cur=$(awk -F'Tempture: ' '{print $2}' <<< $(test-strict-confinement.read-thermal $node 2>&1))
+    printf "Before stress: %0.2f°C\n" $(bc -l <<< "$cur/1000")
+    stress-ng --cpu 8 --io 4 --vm 2 --vm-bytes 128M --fork 4 --timeout 20s
```

Comment:
What is the reason that you provide fixed number for the cpu, io, vm etc ?

Reply:
@hanhsuan I'll restrute this job. Since I've talked with Stanley and we found a way to leverage the current script in contrib/checkbox-ce-oem.